### DISCUSSION TITLE
Single user cannot login

### DISCUSSION THREAD

**Discussion by Grafnus** [outline/outline#5369] [Self Hosting] [answered]
I have a single user that cannot login.

I host my instance of outline with docker using the newest version. I use my own OIDC provider, that works just fine with every other user account. Only one is unable to access the the app. No Error provided in the web ui

In the logs I get this:

```
{
    "error": "User authentication <ID1> already exists for <ID2>, tried to assign to <ID3>",
    "level": "error",
    "message": "Error during authentication",
    "stack": "UnauthorizedError: User authentication <ID1> already exists for <ID2>, tried to assign to <ID3>
    at AuthenticationError (/opt/outline/build/server/errors.js:38:34)
    at accountProvisioner (/opt/outline/build/server/commands/accountProvisioner.js:121:43)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"
}
```

I think that somewhere a cache is already holding a session for this user. Is there a way to clear this cache?

What was tried:

- Restarting the Identity provider (keycloak)
- Restarting outline
- Clearing cookies of outline
- Clearing cookies of keycloak

Any more Ideas?

**Comment by tommoor** [comment, marked as answer]
Your user tried to sign in with an authentication provider ID that is already associated with another user. You have two options:

- Update the email on the user to match the one coming from the authentication provider. (You'll have to use the ID's to find the relevant user)
- Delete the `user_authentication` record associated with the existing user so it can be reassigned to the new one.

**Reply by Grafnus** [reply]
~~For that I need to access the database behind outline. Do I understand that correctly?~~

**Reply by Grafnus** [reply]
Changing the users email address was sufficient.
